src/ram/src/ram.cpp

```diff
@@ -603,7 +563,12 @@ void RamGen::generate(const int bytes_per_word,
                   sites_width);
   }
 
+  int max_y_coord = ram_grid.getHeight() * (word_count + 1);
+  int max_x_coord = ram_grid.getRowWidth();
```

Comment:
Pedantic: move these declarations below `ram_grid.placeGrid()` so that they are closer to where they are used.

Also, mark them as `const`.